script.js

```diff
@@ -18,43 +17,39 @@
         const { icon, description } = data.weather [0];
         const { temp, humidity } = data.main;
         const { speed } = data.wind;
-        /*const { sunrise } = data.sys.sunrise;
-        const { sunset } = data.sys.sunset; are not working*/
-        
-
+        const sunrise = new Date(data.sys.sunrise).toLocaleTimeString('en-US');

```

Comment:
Data.sys.sunrise was a javascript date object so it couldn't be pulled directly off ("destructured" as the { sunrise } syntax does) and used below.